server/spec/test_WAPI.rb

```diff
@@ -309,9 +306,50 @@ def test_renew_token_fail
     h = WAPI.new(a);
 
     r = h.renew_token
-    logger.info "#{__LINE__}: r: "+r.inspect
     assert_equal(401, r.meta_status, "token should not renew")
 
   end
 
+  #### extract correct link from link headers.
+
+  def test_process_link_headers_with_next
+
+    # using mock mostly as a stub.
+    response = MiniTest::Mock.new
+    response.expect :headers, {:link => @link_header_with_next}
+
+    x = @w.process_link_header(response)
+    assert_match /&page=2&/, x, "find existing link header"
```

Comment:
Good question.  Can the test fixtures ensure that enough test data gets loaded to fill at least two pages?